dashboard/test/models/activity_test.rb

```diff
@@ -1,5 +1,6 @@
 require 'test_helper'
 
+require 'async_progress_handler'
```

Comment:
This caused hard to debug problems with the mocha gem recently (mocha needs to be included by minitest not by individual tests because minitest needs to set it up to do unstubbing in teardown). In most cases this won't be a problem because it doesn't matter who is requiring it. In that case, my only concern is that if we actually follow this pattern every test would have a dependency list a mile long (eg. activity test should be requiring user, user_level).